### PR TITLE
resizing ucrreorder

### PR DESCRIPTION
**Pull request by shadrackkibet**
@dannyparsons  i have resized the ucrreoder here. Have a quick check on this so that i can change this on the individual dialogs.

### CODE REVIEW

File: instat/dlgReorderColumns.Designer.vb [hunk 1]
```diff
@@ -39,35 +39,28 @@ Partial Class dlgReorderColumns
     <System.Diagnostics.DebuggerStepThrough()>
     Private Sub InitializeComponent()
         Me.lblColumns = New System.Windows.Forms.Label()
-        Me.ucrReorderColumns = New instat.ucrReorder()
-        Me.ucrBase = New instat.ucrButtons()
+        Me.UcrReorder1 = New instat.ucrReorder()
```
Comment (dannyparsons) [inline instat/dlgReorderColumns.Designer.vb]: Can you set it back to the correct name?

**Comment by dannyparsons** [comment]
Can you send me an image of how the dialog looks now? That will be easiest.

**Comment by shadrackkibet** [comment]
![image](https://user-images.githubusercontent.com/26301973/27829890-821762ac-60cc-11e7-8634-8088d2f9c

@dannyparsons .Here is the size of the control now.

**Comment by shadrackkibet** [comment]
@dannyparsons this now fixes #3588. This is ready for you.

**Comment by shadrackkibet** [comment]
I have also changed the size of the frmmain to fix restore issue here. This now fixes  #2908.

File: instat/dlgReorderColumns.vb [hunk 2]
```diff
@@ -86,7 +86,7 @@ Public Class dlgReorderColumns
         TestOkEnabled()
     End Sub
 
-    Private Sub Controls_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReorderColumns.ControlContentsChanged
+    Private Sub Controls_ControlContentsChanged(ucrChangedControl As ucrCore)
```
Comment (dannyparsons) [inline instat/dlgReorderColumns.vb]: You've removed this event handle, needs to be added back in.

File: instat/dlgReorderDataFrame.vb [hunk 3]
```diff
@@ -80,7 +80,7 @@ Public Class dlgReorderDataFrame
         TestOkEnabled()
     End Sub
 
-    Private Sub Controls_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrDataFrameToReorder.ControlContentsChanged
+    Private Sub Controls_ControlContentsChanged(ucrChangedControl As ucrCore)
```
Comment (dannyparsons) [inline instat/dlgReorderDataFrame.vb]: same here

File: instat/dlgReorderObjects.vb [hunk 4]
```diff
@@ -74,7 +74,7 @@ Public Class dlgReorderObjects
         SetRCode(Me, ucrBase.clsRsyntax.clsBaseFunction, bReset)
     End Sub
 
-    Private Sub ucrReorderObjects_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReorderObjects.ControlContentsChanged
+    Private Sub ucrReorderObjects_ControlContentsChanged(ucrChangedControl As ucrCore)
```
Comment (dannyparsons) [inline instat/dlgReorderObjects.vb]: same here

**Comment by shadrackkibet** [comment]
@dannyparsons i have added back the event handle. This is ready for you.

**Pull request merged**